### PR TITLE
Added some text occurrences in UI

### PR DESCRIPTION
**Pull request by nanu1605**
Added some text occurrences in UI which are not added to `strings.en.yaml` for translation and replaced the occurrences with `getString(' ')` in `chaoscenter/web.` #4147

### CODE REVIEW

**Comment by nanu1605** [comment]
Hello @Saranya-jena, please take a look at the changes.

**Comment by namkyu1999** [comment]
Check DCO, please

**Comment by nanu1605** [comment]
@namkyu1999 Please take a look at this PR

File: chaoscenter/web/src/components/InputSlider/InputSlider.tsx [hunk 1]
```diff
@@ -29,8 +29,8 @@ export default function InputSlider({ initialValue, name, onChange }: InputSlide
       <Layout.Vertical className={cx(css.grow, css.gap1)}>
         <div className={css.rangeSliderCont}>
           <input
-            data-testid="rangeSliderInput"
-            type="range"
+            data-testid={getString('rangesliderinput')}
```
Comment (namkyu1999) [inline chaoscenter/web/src/components/InputSlider/InputSlider.tsx:32]:
```suggestion
            data-testid={getString('rangeSliderInput')}
```

File: chaoscenter/web/src/strings/strings.en.yaml [hunk 2]
```diff
@@ -845,6 +852,8 @@ public: Public
 query: Query
 queryPath: Query path
 ranAt: Ran at
+range: range
+rangesliderinput: rangeSliderInput
```
Comment (namkyu1999) [inline chaoscenter/web/src/strings/strings.en.yaml:856]:
```suggestion
rangeSliderInput: rangeSliderInput
```

File: chaoscenter/web/src/views/AddHubModalWizard/AddHubModalWizard.tsx [hunk 3]
```diff
@@ -260,7 +260,7 @@ const GitConnectionStep: React.FC<
                         PAT
                       </Text>
                     }
-                    placeholder="Enter your Personal Acess Token"
+                    placeholder={getString('EnteryourPersonalAcessToken')}
```
Comment (namkyu1999) [inline chaoscenter/web/src/views/AddHubModalWizard/AddHubModalWizard.tsx:263]:
```suggestion
                    placeholder={getString('enterYourPersonalAccessToken')}
```

File: chaoscenter/web/src/strings/strings.en.yaml [hunk 4]
```diff
@@ -324,6 +328,7 @@ enterProjectName: Enter Project Name
 enterYourEmail: Enter your email
 enterYourName: Enter your name
 enterYourPassword: Enter your password
+EnteryourPersonalAcessToken: Enter your Personal Acess Token
```
Comment (namkyu1999) [inline chaoscenter/web/src/strings/strings.en.yaml:331]:
```suggestion
enterYourPersonalAccessToken: Enter your Personal Access Token
```

File: chaoscenter/web/src/views/AddProbeModalWizard/AddProbeModalWizard.tsx [hunk 5]
```diff
@@ -803,8 +803,8 @@ const TuneDetailsStep: React.FC<
               <>
                 <FormInput.Select
                   name="kubernetesHTTPProperties.method.get.criteria"
-                  label="Criteria"
-                  placeholder="Criteria for data"
+                  label={getString('criteria')}
+                  placeholder={getString('criteriafordata')}
```
Comment (namkyu1999) [inline chaoscenter/web/src/views/AddProbeModalWizard/AddProbeModalWizard.tsx:807]:
```suggestion
                  placeholder={getString('criteriaForData')}
```

File: chaoscenter/web/src/views/AddProbeModalWizard/AddProbeModalWizard.tsx [hunk 6]
```diff
@@ -836,8 +836,8 @@ const TuneDetailsStep: React.FC<
                 />
                 <FormInput.Select
                   name="kubernetesHTTPProperties.method.post.criteria"
-                  label="Criteria"
-                  placeholder="Criteria for data"
+                  label={getString('criteria')}
+                  placeholder={getString('criteriafordata')}
```
Comment (namkyu1999) [inline chaoscenter/web/src/views/AddProbeModalWizard/AddProbeModalWizard.tsx:840]:
```suggestion
                  placeholder={getString('criteriaForData')}
```

File: chaoscenter/web/src/strings/strings.en.yaml [hunk 7]
```diff
@@ -205,6 +207,7 @@ createdAt: Created At
 createdOn: Created On
 creationTime: Creation Time
 criteria: Criteria
+criteriafordata: Criteria for data
```
Comment (namkyu1999) [inline chaoscenter/web/src/strings/strings.en.yaml:210]:
```suggestion
criteriaForData: Criteria for data
```

File: chaoscenter/web/src/views/AddProbeModalWizard/AddProbeModalWizard.tsx [hunk 8]
```diff
@@ -882,8 +882,8 @@ const TuneDetailsStep: React.FC<
             </Text>
             <FormInput.Select
               name="kubernetesCMDProperties.comparator.type"
-              label="Type"
-              placeholder="Type of data"
+              label={getString('type')}
+              placeholder={getString('typeofdata')}
```
Comment (namkyu1999) [inline chaoscenter/web/src/views/AddProbeModalWizard/AddProbeModalWizard.tsx:886]:
```suggestion
              placeholder={getString('typeOfData')}
```

File: chaoscenter/web/src/strings/strings.en.yaml [hunk 9]
```diff
@@ -1076,6 +1087,7 @@ tuneFaultProperties: Tune fault properties
 tuneFaultWeight: Tune fault weightage
 tuneWeight: Tune Weight
 type: Type
+typeofdata: Type of data
```
Comment (namkyu1999) [inline chaoscenter/web/src/strings/strings.en.yaml:1090]:
```suggestion
typeOfData: Type of data
```

File: chaoscenter/web/src/strings/strings.en.yaml [hunk 10]
```diff
@@ -1076,6 +1087,7 @@ tuneFaultProperties: Tune fault properties
...
 udpatePassword: Update Password
```
Comment (namkyu1999) [inline chaoscenter/web/src/strings/strings.en.yaml:1091]:
```suggestion
updatePassword: Update Password
```

can you change it too with relevent files?

File: chaoscenter/web/src/views/AddProbeModalWizard/AddProbeModalWizard.tsx [hunk 11]
```diff
@@ -894,10 +894,10 @@ const TuneDetailsStep: React.FC<
             <Layout.Horizontal flex={{ justifyContent: 'space-between', alignItems: 'flex-start' }} spacing="small">
               <FormInput.Select
                 name="kubernetesCMDProperties.comparator.criteria"
-                label="Comparison Criteria"
+                label={getString('comparisonCriteria')}
                 usePortal
                 style={{ width: '50%' }}
-                placeholder="Criteria for data"
+                placeholder={getString('criteriafordata')}
```
Comment (namkyu1999) [inline chaoscenter/web/src/views/AddProbeModalWizard/AddProbeModalWizard.tsx:900]:
```suggestion
                placeholder={getString('criteriaForData')}
```

File: chaoscenter/web/src/views/AddProbeModalWizard/AddProbeModalWizard.tsx [hunk 12]
```diff
@@ -1018,10 +1018,10 @@ const TuneDetailsStep: React.FC<
         <Layout.Horizontal flex={{ justifyContent: 'space-between', alignItems: 'flex-start' }} spacing="small">
           <FormInput.Select
             name="promProperties.comparator.criteria"
-            label="Comparison Criteria"
+            label={getString('comparisonCriteria')}
             usePortal
             style={{ width: '50%' }}
-            placeholder="Criteria for data"
+            placeholder={getString('criteriafordata')}
```
Comment (namkyu1999) [inline chaoscenter/web/src/views/AddProbeModalWizard/AddProbeModalWizard.tsx:1024]:
```suggestion
            placeholder={getString('criteriaForData')}
```

**Review by namkyu1999 — CHANGES_REQUESTED** [review]
LGTM 🚀
I requested few changes
1. Can you resolve all the issues I mentioned?
2. Can you also make all these changes to `web/src/strings/types.ts`
3. After changes, Can you sort and check strings via `yarn strings:sort` and `yarn strings:check` ?

**Comment by nanu1605** [comment]
Will update the `web/src/strings/types.ts` ASAP

**Comment by nanu1605** [comment]
@namkyu1999 Should I go to folder `/chaoscenter/web` and run `yarn strings`?

**Comment by namkyu1999** [comment]
> @namkyu1999 To regenerate `type.ts` should I go to folder `/chaoscenter/web` and run `yarn strings`?

Hey @nanu1605 , go to the `/chaoscenter/web` and
`yarn strings:check` => check whether your strings are duplicated or not
`yarn strings:sort` => sort strings
`yarn build` => this is what you want

I recommend you doing all of them :)

**Comment by nanu1605** [comment]
@namkyu1999  Please take a look at this PR :)

**Review by namkyu1999 — APPROVED** [review]
LGTM 🚀🚀

**Review by Saranya-jena — APPROVED** [review]

**Review by namkyu1999 — APPROVED** [review]

**Review by amityt — APPROVED** [review]

**Review by SahilKr24 — APPROVED** [review]

**Pull request merged**